ssz/src/main/java/tech/pegasys/teku/ssz/schema/SszPrimitiveSchemas.java

```diff
@@ -142,6 +142,29 @@ public TreeNode updateBackingNode(TreeNode srcNode, int index, SszData newValue)
           return LeafNode.create(newBytes);
         }
 
+        @Override
+        public TreeNode updateBackingNode(
+            TreeNode srcNode, int[] internalIndexes, SszData[] newValues, int count) {
```

Comment:
Should we be validating these params here?

Reply:
Not relevant any more